google-cloud-bigtable-stats/src/main/java/com/google/cloud/bigtable/stats/BigtableStackdriverExportUtils.java

```diff
@@ -102,25 +104,71 @@ class BigtableStackdriverExportUtils {
   private static final LabelKey CLIENT_UID_LABEL_KEY =
       LabelKey.create(BuiltinMeasureConstants.CLIENT_UID.getName(), "client uid");
 
+  static boolean isBigtableTableMetric(MetricDescriptor metricDescriptor) {
+    return !metricDescriptor.getName().equals(PER_CONNECTION_ERROR_COUNT_VIEW.getName().asString());
+  }
+
+  static boolean shouldExportMetric(MetricDescriptor metricDescriptor) {
+    return metricDescriptor.getName().contains("bigtable")
```

Comment:
this can be refactored to
return isBigtableTableMetric() || (metricDescriptor.getName().equals(PER_CONNECTION_ERROR_COUNT_VIEW.getName().asString()) && (isEnvGce() || isEnvGke())